### PR TITLE
NA-22/Add-material-UI

### PR DESCRIPTION
**Pull request by yevgeny-semak**

### CODE REVIEW

File: public/index.css [hunk 1]
```diff
@@ -26,15 +27,14 @@ body {
 }
 
 .search-bar {
-  position:relative;
   width: 40rem;
-  margin: 0 1rem;
+  height: 3rem;
+  margin: 0 5rem;
   background: #fff;
```
Comment (veranika-zakharchenia) [inline public/index.css:33]: it is better to move all colors codes to constants

File: public/index.css [hunk 2]
```diff
@@ -26,15 +27,14 @@ body {
...
   padding: 15px;
```
Comment (veranika-zakharchenia) [inline public/index.css:34]: try to avoid `px`

File: public/index.css [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  width: 100%;
+  border: none;
+  padding: 4px;
+  outline: none;
+  font-size: 1.2em;
+  font-family: inherit;
+  resize: none;
+}
+form.note-add button {
+  position: absolute;
```
Comment (veranika-zakharchenia) [inline public/index.css:93]: is this exactly the only solution?

File: public/index.css [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+  right: 18px;
+  bottom: -18px;
```
Comment (veranika-zakharchenia) [inline public/index.css:95]: same here. try to avoid -

**Review by veranika-zakharchenia — CHANGES_REQUESTED** [review]
css changes are requested

File: public/index.css (hunk 3 above)
Comment (yevgeny-semak) [reply public/index.css:93]: Used position: relative and float:right instead

File: public/index.css (hunk 4 above)
Comment (yevgeny-semak) [reply public/index.css:95]: fixed

File: public/index.css (hunk 2 above)
Comment (yevgeny-semak) [reply public/index.css:34]: fixed

File: public/index.css (hunk 1 above)
Comment (yevgeny-semak) [reply public/index.css:33]: to be redone later

**Pull request merged**